content/docs/1.1.0/deploy/install/_index.md

```diff
@@ -46,7 +46,7 @@ For the minimum recommended hardware, refer to the [best practices guide.](../..
 
 ### Using the Environment Check Script
 
-We've written a script to help you gather enough information about the factors.
+We've written a script to help you gather enough information about the factors. NB: Run this only in the control plane node.
```

Comment:
What is NB means?

Reply:
Ah, it is nota bene meaning note well. We can remove it so people does not confuse :D.

Reply:
Would change to "Note" better?

Reply:
Sure!